### PR TITLE
Check packaging, add missing file, add info about versioning

### PR DESCRIPTION
**Pull request by fabiencelier**
- Add info in the readme about versioning
- check the packaging in wheel and conda in the test workflow
- add the missing file in the wheel / conda package

### CODE REVIEW

**Comment by fabiencelier** [comment]
I have added a new job with duplicate steps but I can't find how to avoid duplicate steps.
I have open https://github.com/atoti/jdk4py/issues/23

File: README.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ conda config --add channels https://conda.atoti.io
 conda install jdk4py
 ```
 
+## Versioning
+
+jdk4py versions contains 4 figures:
+
+  - The first 3 figures is the Java version
```
Comment (bloussou) [inline README.md:26]:
```suggestion
  - The first 3 figures are the Java version
```

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -46,3 +46,45 @@ jobs:
...
+          JDK4PY_BUILD_PLATFORM: ubuntu-latest
+        run: poetry run python setup.py bdist_wheel
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          channels: conda-forge
+          conda-build-version: 3.20.0
+
+      - name: Build Conda package on ubuntu and latest Python
+        run: conda build --output-folder dist . --no-anaconda-upload --debug
```
Comment (bloussou) [inline .github/workflows/test.yaml:90]: Missing empty line at the end of the file ? (not sure if it is usefull but github always highlight that)

File: README.md [hunk 3]
```diff
@@ -19,6 +19,13 @@ conda config --add channels https://conda.atoti.io
 conda install jdk4py
 ```
 
+## Versioning
+
+jdk4py versions contains 4 figures:
```
Comment (bloussou) [inline README.md:24]:
```suggestion
jdk4py version contains 4 figures:
```

**Review by bloussou — COMMENTED** [review]
Small typos I think :)

**Review by bloussou — APPROVED** [review]

**Pull request merged**